example/hyprland.conf

```diff
@@ -1,88 +1,117 @@
 # This is an example Hyprland config file.
-#
 # Refer to the wiki for more information.
+# https://wiki.hyprland.org/Configuring/Configuring-Hyprland/
 
-#
 # Please note not all available settings / options are set here.
 # For a full list, see the wiki
-#
+
+# You can split this configuration into multiple files
+# Create your files separately and then link them to this file like this:
+# source = ~/.config/hypr/myColors.conf
+
+
+################
+### MONITORS ###
+################
 
 # See https://wiki.hyprland.org/Configuring/Monitors/
 monitor=,preferred,auto,auto
 
 
-# See https://wiki.hyprland.org/Configuring/Keywords/ for more
-
-# Execute your favorite apps at launch
-# exec-once = waybar & hyprpaper & firefox
+###################
+### MY PROGRAMS ###
+###################
 
-# Source a file (multi-file configs)
-# source = ~/.config/hypr/myColors.conf
+# See https://wiki.hyprland.org/Configuring/Keywords/
 
 # Set programs that you use
 $terminal = kitty
 $fileManager = dolphin
 $menu = wofi --show drun
 
-# Some default env vars.
-env = XCURSOR_SIZE,24
-env = QT_QPA_PLATFORMTHEME,qt5ct # change to qt6ct if you have that
 
-# For all categories, see https://wiki.hyprland.org/Configuring/Variables/
-input {
-    kb_layout = us
-    kb_variant =
-    kb_model =
-    kb_options =
-    kb_rules =
+#################
+### AUTOSTART ###
+#################
 
-    follow_mouse = 1
+# Autostart necessary processes (like notifications daemons, status bars, etc.)
+# Or execute your favorite apps at launch like this:
 
-    touchpad {
-        natural_scroll = false
-    }
+# exec-once = $terminal
+# exec-once = nm-applet &
+# exec-once = waybar & hyprpaper & firefox
 
-    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
-}
 
-general {
-    # See https://wiki.hyprland.org/Configuring/Variables/ for more
+#############################
+### ENVIRONMENT VARIABLES ###
+#############################
+
+# See https://wiki.hyprland.org/Configuring/Environment-variables/
 
+env = XCURSOR_SIZE,24
+env = QT_QPA_PLATFORMTHEME,qt5ct # change to qt6ct if you have that
```

Comment:
I'd rather not have this one, people usually set it in /etc/environment etc

Reply:
Ok, I can remove it. It's present in the default config, so i just left it there.